Release notes for CouchbaseNetClient 3.2.5

RiPont opened the pull request.

RiPont committed:
Release notes for CouchbaseNetClient 3.2.5

jeffrymorris commented on modules/project-docs/pages/sdk-release-notes.adoc:
```diff
@@ -107,6 +107,46 @@ git pull origin master
+
+Version 3.2.5 is the fifth release of the 3.2 series.
+
+https://packages.couchbase.com/clients/net/3.3/Couchbase-Net-Client-3.2.5.zip[Download] |
```
Need read permissions set for this asset: <Code>AccessDenied</Code>

jeffrymorris replied:
Link should be https://packages.couchbase.com/clients/net/3.2/Couchbase-Net-Client-3.2.5.zip

jeffrymorris commented on modules/project-docs/pages/sdk-release-notes.adoc:
```diff
@@ -107,6 +107,46 @@ git pull origin master
+
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/NCBC-2851[NCBC-2851]: TimeoutExceptions continue after rebound in Failover/Eject tests.
```
Docs team has asked that we do not simply copy the title from Jira but provide more user friendly, easy to read description. Maybe this is something we do in a separate commit though as this doc as is still adds value.

jeffrymorris reviewed (approved):
Approving, but please read my notes.

osfameron commented:
Looks good.
I'll do some gardening on the wording ( in progress but I gotta go eat now, so will finish tomorrow)

As @jeffrymorris mentioned, the .zip link needs update before we can merge in any case!

RiPont committed:
Fix ZIP file URLs.

osfameron committed:
Gardening 3.2.5

Ticket descriptions in past tense, from customer perspective
(NB: often the Gerrit summary is a better starting point than the Jira ticket)

Removed one ticket that clearly stated "no behavioral change" as relnotes only
for consumer visible changes.

(There are a few tidy/target tickets that are retained as they also included
bugfixes and e.g. .NET versions would be of interest to developers, so I
couldn't make a call on those.)

Grouped a few more tickets together.

Added spacing to make it easier to read/edit.

osfameron reviewed (approved):
This looks good thanks @RiPont @jeffrymorris 

I've made some additional edits, mostly based on Gerrit summaries (which generally have a better user-focused description than the original Jira)

Please do look through those, to sense check if I've introduced any errors.
